etc/profile-a-l/email-common.profile

```diff
@@ -40,7 +41,9 @@ whitelist ${HOME}/Mail
 whitelist ${RUNUSER}/gnupg
 whitelist /usr/share/gnupg
 whitelist /usr/share/gnupg2
+whitelist /var/lib/clamav 
 whitelist /var/mail
+whitelist /var/run/clamav/clamd.ctl
```

Comment:
A whitelisting of /var/run/clamav suffices here.

Reply:
Changed